src/app/modules/game/pages/lobby/lobby.component.scss

```diff
@@ -1,9 +1,117 @@
+@import "plugins/variables";
+@import "plugins/mixins";
+
 .lobby {
+  width: 100%;
+  &__container {
+    background-image: url(/assets/backgrounds/lobby-background.gif);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-attachment: fixed;
+    top: 0;
+    margin: 0;
+    padding: 0;
+    background-size: 90% 100%;
+    height: 100vh;
+    position: relative;
+  }
+  &__back {
+    padding: 0.2rem 1rem;
+    cursor: pointer;
+    width: 100px;
+    &:hover {
+      color: $primary-color--dark;
+    }
+  }
+  &__sidebar {
+    width: 20%;
+    top: 0;
+    right: 0;
+    min-height: 100vh;
+    z-index: 1;
+    background-color: $primary-color--dark;
+    border-left: 3px solid $black;
+    position: absolute;
+  }
+  &__chat {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    margin-left: 0.5rem;
+  }
   &__users {
     display: flex;
+    flex-direction: column;
     list-style-type: none;
+    justify-content: space-around;
+    text-align: center;
+    margin: 3rem auto;
+  }
+  &__button {
+    text-align: center;
+    background-color: $white;
+    font-weight: bold;
+    font-size: 4rem;
+    border-radius: 10px;
+    margin: 1rem;
+    padding: 0.5rem 1rem;
```

Comment:
same

Reply:
Agreed with @maxonchil to leave as is. Relative units are more flexible in responsive design.